tests/cypress/utils.js

```diff
@@ -58,6 +58,8 @@ function injectHtmlAndBootAlpine(cy, templateAndPotentiallyScripts, callback, pa
 
         el.evalScripts(scripts)
 
+        console.info(scripts)
```

Comment:
Debug print

Reply:
Whoops… I was trying to figure it out what was happening.

Thanks for the head's up!